classes/class-wc-qd-invoice-manager.php

```diff
@@ -233,6 +235,34 @@ public function get_payment_method( $order_id ) {
 		}
 		return $method;
 	}
+
+  /**
+   * Get payment processor for Quaderno
+   *
+   * @param $order_id
+   */
+  public function get_payment_processor( $order_id ) {
+    $payment_id = get_post_meta( $order_id, '_payment_method', true );
+    $method = '';
+    switch( $payment_id ) {
+      case 'paypal':
+        $method = 'paypal';
+        break;
+      case 'stripe':
+        $method = 'stripe';
+        break;
+    }
+    return $method;
+  }
+
+  /**
+   * Get payment processor id for Quaderno
+   *
+   * @param $order_id
+   */
+  public function get_payment_processor_id( $order_id ) {
+    return get_post_meta( $order_id, '_transaction_id', true );
```

Comment:
Is it necessary a whole function for this? We can just use the `get_post_meta` function in line 38, right?